From 42c911d5eb0c6b59cf3b3739d138e1d766fe5561 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Tue, 20 Nov 2018 16:02:24 +0800
Subject: [PATCH 5157/5242] MLK-20417-2: ARM64: dts: disable intmux in imx8qm
 m4 dts

commit  3a30144862f8f683ccc6b350b363f8287dcc0b80 from
https://source.codeaurora.org/external/imx/linux-imx.git

disable intmux in m4 dts for it is component in m4 domain

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit 252831d4b0699c9ee609613a7c00ab8f77bfbeb9)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8qm-mek-m4.dts |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-m4.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-m4.dts
index c0eef8a..3c2f664 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-m4.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-m4.dts
@@ -56,6 +56,14 @@
 	status = "disabled";
 };
 
+&intmux_cm41 {
+	status = "disabled";
+};
+
+&intmux_cm40 {
+	status = "disabled";
+};
+
 &flexcan1 {
 	status = "disabled";
 };
-- 
1.7.9.5

